runtime.txt

```diff
@@ -1 +1,3 @@
 python-3.11.1
+webp
+cmake
```

Comment:
Heroku doesn't understand these lines, so I think they can be removed.

```
-----> Using Python version specified in runtime.txt
 !     Requested runtime 'python-3.11.1
webp
cmake' is not available for this stack (heroku-22).
```

Reply:
Done! Curious what the outcome is :)